### PR TITLE
New Port Rebound

### PR DESCRIPTION
**Pull request by Tlaggypyro**
# New Port for { Rebound }

## Game Information
- **Title**: { Rebound }
- **URL**: { https://rebound.itch.io/rebound }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/rebound/screenshot.jpg
Comment (Cebion) [inline ports/rebound/screenshot.jpg:1]: Screenshot needs to be not squashed and 640x480 in resolution.
You can take screenshots on your device via https://portmaster.games/port-tools.html#capture-video-and-screenshots for example

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/rebound/Rebound.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+get_controls
+
+GAMEDIR=/$directory/ports/rebound
+CONFDIR="$GAMEDIR/conf/"
+
+CUR_TTY=/dev/tty0
+$ESUDO chmod 666 $CUR_TTY
+
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
```
Comment (kloptops) [inline ports/rebound/Rebound.sh:26]: Old logging style, use this instead: `> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1`

File: ports/rebound/port.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": 2,
+    "name": "rebound.zip",
+    "items": [
+        "Rebound.sh",
+        "rebound"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Brad Has A Pain",
```
Comment (kloptops) [inline ports/rebound/port.json:10]: Is the game named rebound, or brad had a pain?

**Review by kloptops — CHANGES_REQUESTED** [review]
Just need fixes on the `port.json` and `Rebound.sh`

**Review by Cebion — APPROVED** [review]

**Pull request merged**